docs/app/pages/components/components-sections/hierarchy-bar/hierarchy-bar/snippets/app.ts

```diff
@@ -26,26 +25,60 @@ export class AppComponent {
 
             observer.next([
               {
-                icon: 'https://uxaspects.github.io/UXAspects/assets/IconManagerColorized.png',
```

Comment:
Seems good, just need to fix indentation in this file. If this was done to avoid line wrapping, perhaps variables can be used for the URLs instead?